dom0/sd-whonix-hidserv-key.sls

```diff
@@ -4,7 +4,24 @@
 {% import_json "sd/config.json" as d %}
 
 # add hidden service auth key to torrc
+{% if d.hidservv3.hostname == "ADD_YOUR_VALUE" %}
 sd-whonix-hidserv-key:
   file.append:
     - name: /usr/local/etc/torrc.d/50_user.conf
-    - text: HidServAuth {{ d.hidserv.hostname }} {{ d.hidserv.key }}
+    - text: HidServAuth {{ d.hidserv.hostname }}.onion {{ d.hidserv.key }}
+{% else %}
+sd-whonix-hidservv3-directory-path:
+  file.append:
+    - name: /usr/local/etc/torrc.d/50_user.conf
+    - text: ClientOnionAuthDir /var/lib/tor/keys
+
+install-sd-whonix-tor-private-key:
+  file.managed:
+    - name: /var/lib/tor/keys/app-journalist.auth_private
+    - source: salt://sd/sd-whonix/app-journalist.yaml
+    - template: jinja
+    - context:
+        hostname: {{ d.hidservv3.hostname }}
+        key: {{ d.hidservv3.key }}
+    - mode: 0600
```

Comment:
nit: since the permissions are 600 here it might be best to specify owner/group similar to https://github.com/freedomofpress/securedrop-workstation/blob/f4ae1e4d9de46092b77dcb16d4f4d940792758ac/dom0/sd-gpg-files.sls#L32-L33